datafusion/functions/Cargo.toml

```diff
@@ -47,9 +49,23 @@ path = "src/lib.rs"
 
 [dependencies]
 arrow = { workspace = true }
+arrow-array = { workspace = true }
 base64 = { version = "0.21", optional = true }
+chrono = { workspace = true }
 datafusion-common = { workspace = true }
 datafusion-execution = { workspace = true }
 datafusion-expr = { workspace = true }
+datafusion-physical-expr = { workspace = true }
```

Comment:
I also think this dependency is uneeded. I will  pull it out into a common function in `ColumnarValue` in